src/starter/files/src/app/shared/shared.module.ts

```diff
@@ -7,26 +7,25 @@ import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatInputModule } from '@angular/material/input';
 import { MatButtonModule } from '@angular/material/button';
 import { ReactiveFormsModule } from '@angular/forms';
-
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
Se `CUSTOM_ELEMENTS_SCHEMA` se for usado neste módulo, o import deve ser removido.